<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddService</class>
 <widget class="QMainWindow" name="AddService">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>220</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>220</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Додати послугу</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Назва послуги</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Ціна послуги</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Примітка</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="ServiceTitleLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Вид послуги</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QPushButton" name="AddServiceButton">
      <property name="text">
       <string>Додати</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="CommentaryTextEdit"/>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="PriceSpinBox">
      <property name="maximum">
       <number>999999</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="ServiceType">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
